### PR TITLE
Feature/general update sass

### PR DESCRIPTION
**Pull request by tylrs**
## Description

- Adds styling to all pages including layout, font sizing, colors, etc.
- Still needs accessibility features 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactor

## Checklist:
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -9,24 +9,24 @@
...
   </header>
   <main>
     <section class="login-view" id="loginView">
       <form class="login-container" id="loginContainer">
-        <h2>Login To Your Account</h2>
-        <label for="userName">UserName</label>
-        <input type="text" name="" value="" id="userNameInput" required>
+        <h2 class="login-title">The Overlook Hotel</h2>
+        <label for="userName">Username</label>
+        <input type="text" name="" value="customer25" id="userNameInput" required>
```
Comment (alexmkio) [inline dist/index.html:19]: Such a good idea hardcoding those values during buildout

File: src/css/base.scss [hunk 2]
```diff
@@ -2,6 +2,10 @@
 $primary-background: white;
 @import '_reset.scss';
 @import 'flatpickr/dist/flatpickr.css';
+@import '_variables.scss';
+@import '_text.scss';
+@import '_buttons.scss';
+@import '_cards.scss';
 
 body {
```
Comment (alexmkio) [inline src/css/base.scss:10]: your base.scss or index.scss or *whatever* webpack is being directed to in index.html or scripts.js should *just* have your imports I believe.
Search "Generally, your structure will have the following" [here](https://frontend.turing.edu/lessons/module-2/introduction-to-sass.html)

File: src/domUpdates.js [hunk 3]
```diff
@@ -6,17 +6,25 @@ let domUpdates = {
...
-          <h5>${booking.date}</h4>
-          <p>${booking.roomType}</p>
-          <p>Room Number ${booking.roomNumber}</p>
-          <p>Beds: ${booking.numBeds} ${booking.bedSize}<p>
-          <p>$${booking.costPerNight}/ night</p>
-          <p>Bidet included?</p>
-          <p>${bidetMessage}</p>
+          <div class="card-header">
+            <h4>Your Booking On:</h4>
+            <h4>${booking.date}</h4>
```
Comment (alexmkio) [inline src/domUpdates.js:15]: this heading level should be an H5

File: src/domUpdates.js [hunk 4]
```diff
@@ -6,17 +6,25 @@ let domUpdates = {
...
       `
         <article class="booking-card">
-          <h5>${booking.date}</h4>
-          <p>${booking.roomType}</p>
-          <p>Room Number ${booking.roomNumber}</p>
-          <p>Beds: ${booking.numBeds} ${booking.bedSize}<p>
-          <p>$${booking.costPerNight}/ night</p>
-          <p>Bidet included?</p>
-          <p>${bidetMessage}</p>
+          <div class="card-header">
```
Comment (alexmkio) [inline src/domUpdates.js:13]: I couldn't tab through your cards. As Ash said, look into "tabindex=0".

File: src/domUpdates.js [hunk 5]
```diff
@@ -41,18 +49,37 @@ let domUpdates = {
...
-        <p>Room Number ${room.number}</p>
-        <p>Beds: ${room.numBeds} ${room.bedSize}<p>
-        <p>$${room.costPerNight}/ night</p>
-        <p>Bidet included?</p>
-        <p>${bidetMessage}</p>
-      </article>
+        <article class="room-card" id="${room.number}">
+          <div class="card-header">
+            <h4>Date Available:</h4>
+            <h4>${searchDate}</h4>
```
Comment (alexmkio) [inline src/domUpdates.js:69]: Should be an H5, but I'd look into dl, dt, dd semantics.

File: src/domUpdates.js [hunk 6]
```diff
@@ -41,18 +49,37 @@ let domUpdates = {
...
+          </div>
+          <div class="card-details">
+            <div class="booking-info">
+              <p>Room #${room.number}</p>
```
Comment (alexmkio) [inline src/domUpdates.js:73]: very divvy. look into sections and dl, dt, dd semantics

**Review by alexmkio — COMMENTED** [review]
FYI there is a contrast issue on the total spent amount. I think it would pass the wave test if the text was larger.

Also, looks like you're still developing for desktop layout but FYI there are some constraint issues with the cards in mobile.

File: dist/index.html (hunk 1 above)
Comment (tylrs) [reply dist/index.html:19]: Yep makes it easy to deal with

File: src/css/base.scss (hunk 2 above)
Comment (tylrs) [reply src/css/base.scss:10]: I think the /*...*/ part of the sample structure is where the rest of your base scss code goes. Here's the part of the lesson plan I believe you are referencing:

@import 'Header';
@import 'Button';
@import 'Author';
@import 'Story';
@import 'Stories';
/*...*/
/*...*/

File: src/domUpdates.js (hunk 4 above)
Comment (tylrs) [reply src/domUpdates.js:13]: yes good idea thank you!

File: src/domUpdates.js (hunk 6 above)
Comment (tylrs) [reply src/domUpdates.js:73]: Agreed yes I'll try to take them out!

**Pull request merged**